### ISSUE TITLE
Venom doesn't filter HEVC

### ISSUE THREAD

**Issue by diddy81** [123Venom/plugin.video.venom#10]
Line 1146 in sources.py 
```

        try: 
            if not HEVC == 'true':
                self.sources = [i for i in self.sources if 'label' or 'multiline_label' in i['label']]
        except:
            pass

        # self.sources = [i for i in self.sources if 'label' or 'multiline_label' in i]
        self.sources = [i for i in self.sources if 'label' in i]

```
should be like this to filter HEVC and any file labeled X265 instead of HEVC
```
        try:
            if not HEVC == 'true':
                self.sources = [i for i in self.sources if not 'HEVC' in i['label']]
                self.sources = [i for i in self.sources if not 'X265' in i['label']]
        except Exception:
            pass

        self.sources = [i for i in self.sources if 'label' or 'multiline_label' in i['label']]
        # self.sources = [i for i in self.sources if 'label' in i]
```

**Comment by 123Venom** [comment]
I'll take a look but give me a couple days...knee deep and working right now to fulfill a business deadline that's got my ass in hot water..lol.

**Comment by 123Venom** [comment]
Curiosity got the better of me because even looking at that first line I can clearly see it's wrong.  I had to back up a few version cause I swear it was once right.  Seems the version I added multiline_label I seriously blew that and broke it.  I fixed it and added your new line that I never had for 'X265'.  Appreciate your time man!!!

**Comment by 123Venom** [comment]
I want to run a change by you to see your opinion.  I noticed with the new HEVC filter working that the link list numbering is now out to lunch.  Makes sense given that it falls after the source list is numbered so it removes numbered items.  I can only predict another user complaint coming soon once discovered, so I'd like to nip it now.  The attached I moved up before numbering and label assignment.  This did pose a problem with the label not being available to us so I used the url and added yet a few more check cases I noted when logged.  This seems to be working well and keeps the numbering in check also.  Let me know your thoughts before I push in the next update I'm trying to wrap up.

![Capture - Copy](https://user-images.githubusercontent.com/46383815/61437781-e304fc80-a90b-11e9-8f52-e00986e0724b.PNG)
